packages/fast-components-react-base/src/breadcrumb/README.md

```diff
@@ -0,0 +1,8 @@
+## Breadcrumb
+A generic breadcrumb component following the w3.org [breadcrumb](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html) pattern. The *breadcrumb* will accept any child elements but is built to work with `a` elements.
+
+### Usage
+To denote the current page the *breadcrumb* automatically sets the last child `aria-current` prop to "page". Addtionally *breadcrumb* accepts an optional `separator` prop that will render a ReactNode between navigation elements. To support `rtl` in jss you need to target the `breadcrumb_ol` class for *breadcrumb* and set it to `display: "flex", flexWrap: "wrap"`

```

Comment:
> To support `rtl` in jss you need to target the `breadcrumb_ol` class for *breadcrumb* and set it to `display: "flex", flexWrap: "wrap"`

I don't think we need to include this. It's not fully accurate as this is *one way* and very specific to our MSFT implementation. There are a bunch of ways we could address RTL scenarios; for MSFT we are using flexbox, but other users may not choose to leverage that technology.